config-linux.md

```diff
@@ -635,6 +635,7 @@ The following parameters can be specified to set up seccomp:
 
 **`maskedPaths`** (array of strings, OPTIONAL) will mask over the provided paths inside the container so that they cannot be read.
     The values MUST be absolute paths in the [container namespace](glossary.md#container_namespace).
+    Unexistent paths MUST be skipped without generating an error.
```

Comment:
"Unexistent" is not a word. Either "inexistent" or "nonexistent".
And this is best a `SHOULD`, not a `MUST`.